app/src/main/res/layout/fragment_file_complaint.xml

```diff
@@ -296,15 +296,6 @@
             android:weightSum="2"
             android:layout_gravity="bottom">
 
-            <Button
-                android:id="@+id/button_analysis"
-                android:layout_width="match_parent"
-                android:layout_height="50dp"
-                android:layout_weight="1"
-                android:background="@color/colorPrimary"
-                android:text="ANALYSIS"
-                android:textColor="@color/colorWhite" />
```

Comment:
I have no idea what this was for, but hope it was intenetional

Reply:
I'm not exactly sure about this either as this was present in the original design. But I think the original idea was that clicking on this button would provide some kind of analysis on the complaint. This information was supposed to come from a ML model (a functionality to be added when the app was updated). I think the idea was dropped in favor of the relevant complaints functionality.